x-pack/plugins/alerting/common/routes/maintenance_window/apis/find/schemas/v1.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { maintenanceWindowResponseSchemaV1 } from '../../../response';
+
+export const findMaintenanceWindowsRequestQuerySchema = schema.object({
+  page: schema.maybe(
+    schema.number({
+      defaultValue: 1,
+      min: 1,
+      meta: {
+        description: 'The page number to return.',
+      },
+    })
+  ),
+  per_page: schema.maybe(
```

Comment:
Let's also add `max :100`.

Reply:
Make sense. Did not do it because our rule fine API does not have it.